lib/kubeclient/common.rb

```diff
@@ -87,11 +89,25 @@ def initialize_client(
       @http_max_redirects = http_max_redirects
       @as = as
 
+      @log = Logger.new(STDOUT)
+      @log.formatter = proc do |severity, datetime, progname, msg|
+          "#{datetime} [#{severity}]: #{msg}\n"
+      end
+
       if auth_options[:bearer_token]
         bearer_token(@auth_options[:bearer_token])
-      elsif auth_options[:bearer_token_file]
+      end
```

Comment:
I wonder whether this `end` should be `elsif`.  Specifically, if one runs inside a cluster and passes an explicit `bearer_token: ...`, the auto-detected `DEFAULT_BEARER_TOKEN_FILE` will override it?

Reply:
Yes, it would override it. I have removed auto-detection so users need to manually provide path to enable refreshing the token